zeppelin-web-angular/src/app/pages/workspace/share/result/result.component.ts

```diff
@@ -26,7 +26,7 @@ import {
   ViewContainerRef
 } from '@angular/core';
 import { DomSanitizer, SafeHtml, SafeUrl } from '@angular/platform-browser';
-import { default as AnsiUp } from 'ansi_up';
+import { AnsiUp } from 'ansi_up';
```

Comment:
<img width="387" height="84" alt="Image" src="https://github.com/user-attachments/assets/e46013de-1924-435a-b184-15ea5632a269" />
It seems like type definitions and related setup for **ansi_up** are required.

Reply:
I can’t reproduce this locally. ansi_up@6.0.6 ships its own type definitions. Could you try again on your end? If it still fails, a clean install (remove node_modules and the lockfile, then reinstall) or sharing your TS version and the exact error would help.